app/views/userProfile.scala.html

```diff
@@ -14,25 +15,28 @@
             <div class="row" id="profile-container">
                 <div id="username">@user.get.username</div>
                 <div id="org-container">
-                    @if(!UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
-                        <div id="org-name">@OrganizationTable.getOrganizationName(UserOrgTable.getAllOrgs(user.get.userId).head)</div>
-                    } else {
-                        <div id="org-name">@Messages("dashboard.no.org")</div>
-                    }
-                    <div id="edit-org-dropdown" class="dropdown" data-toggle="popover" data-trigger="hover" data-placement="right" data-content="@Messages("dashboard.edit.info")">
-                        <a id="edit-org-button" role="button" data-toggle="dropdown" href="#">@Messages("dashboard.edit.org") <b class="caret"></b></a>
-                        <ul id="edit-org-menu" class="dropdown-menu" role="menu">
-                            @if(OrganizationTable.getAllOrganizations.isEmpty) {
-                                <li class="menu-info">@Messages("dashboard.orgs.unavailable")</li>
+                    <div id="edit-org-dropdown" class="dropdown">
+                        <a id="edit-org-button" role="button" data-toggle="dropdown" href="#">
+                            @if(userOrg.nonEmpty) {
+                                @Messages("dashboard.current.team.greeting", OrganizationTable.getOrganizationName(userOrg.headOption.getOrElse(-1)).getOrElse("Team Name Not Found")) <b class="caret"></b>
+                            } else {
+                                @Messages("dashboard.join.team") <b class="caret"></b>
                             }
-                            @if(!UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
+                        </a>
+                        <ul id="edit-org-menu" class="dropdown-menu" role="menu">
+                            @if(userOrg.nonEmpty) {
                                 <li>
-                                    <a id="from-@UserOrgTable.getAllOrgs(user.get.userId).headOption.getOrElse(0)-to-0" role="menuitem" class="put-user-org">@Messages("dashboard.leave.org")</a>
+                                    <a id="from-@UserOrgTable.getOrg(user.get.userId).headOption.getOrElse(0)-to-0" role="menuitem" class="put-user-org">
+                                        <strong>@Messages("dashboard.leave.team", OrganizationTable.getOrganizationName(userOrg.headOption.getOrElse(-1)).getOrElse("Team Name Not Found"))</strong>
+                                    </a>
                                 </li>
                             }
+                            <li>
+                                <a id="create-org-button" role="menuitem" class="create-org" data-toggle="modal" data-target="#create-team-modal" href="#"><Strong>@Messages("dashboard.create.team")</Strong></a>
```

Comment:
Let's switch to lowercase "strong"!

Reply:
sorry, this must have been a change lost in the dev process. fixed